doc/distributed_python/usage.rst

```diff
@@ -1,2 +1,33 @@
 Creating and running UDFs
--------------------------
\ No newline at end of file
+-------------------------
+
+For best performance, we recommend that you create a script using CREATE SCRIPT and then use this script within a SELECT statement. Embedding the script within SQL in this way provides the best performance and scalability.
```

Comment:
Sorry, I should have picked it in the first pass. I don't know any other way of creating a UDF other than executing the `CREATE SCRIPT` command. These two sentences do not make a lot of sense to me.